BSP_development_guide.md

```diff
@@ -48,28 +48,35 @@ esp_err_t bsp_i2c_deinit(void);
 * See any BSP
 
 #### Display
-* BSPs with display are shipped with [LVGL](https://components.espressif.com/components/lvgl/lvgl) component
-* Display functions are divided into two parts. The first part is only HW initialization (include\bsp\display.h) and second part is initialization of the LVGL (BSP header).
-* Functions, which must be defined in BSP *.c file:
+* BSPs with display are shipped with [LVGL](https://components.espressif.com/components/lvgl/lvgl) component by default. For version without LVGL, you can use BSP version with `-noglib` suffix (eg. `2.1.0-noglib`)
```

Comment:
How would it be distributed in component manager?

Reply:
As discussed: The distribution will use suffix to BSP _name_.

So there will be derived component (linked to the original) with name eg `esp32_s3_eye_noglib`